api/hello_test.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
```

Comment:
Add context to fix failing unit test:

`var ctx = context.Background()`

Reply:
as above